common.blocks/menu-item/menu-item.deps.js

```diff
@@ -5,4 +5,10 @@
 {
     tech : 'spec.js',
     mustDeps : { tech : 'bemhtml', block : 'menu-item' }
+},
+{
+    tech : 'tmpl-spec.js',
+    mustDeps : [
+        { tech : 'bemhtml', block : 'link' }
```

Comment:
without these deps templates for block `link` will not be applied in tmpl-specs.